mara_description/urdf/mara_gazebo.urdf.xacro

```diff
@@ -15,14 +15,11 @@
     		<joint_name>motor4</joint_name>
     		<joint_name>motor5</joint_name>
         <joint_name>motor6</joint_name>
-        <joint_name>left_inner_knuckle_joint</joint_name>
-        <joint_name>left_outer_knuckle_joint</joint_name>
-        <joint_name>left_outer_finger_joint</joint_name>
-        <joint_name>left_inner_finger_joint</joint_name>
-        <joint_name>right_inner_knuckle_joint</joint_name>
-        <joint_name>right_outer_knuckle_joint</joint_name>
-        <joint_name>right_outer_finger_joint</joint_name>
-        <joint_name>right_inner_finger_joint</joint_name>
+        <joint_name>robotiq_arg2f_base_to_left_inner_knuckle</joint_name>
```

Comment:
I think we are using mimic, I think we should use`joint_finger` instead all of these joints

Reply:
I followed the old structure. I can replace all of them with joint_finger and test it.

Reply:
You are right, in MARA repo is already like that, just with the `joint_finger`.